shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/TripInstantDisplay.kt

```diff
@@ -154,4 +154,6 @@ sealed class TripInstantDisplay {
             return Minutes(minutes)
         }
     }
+
+    fun scheduleBasedDisplay() {}
```

Comment:
```suggestion
```

Reply:
TY! I started thinking about the refactor and forgot to back it all out